File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n=nums.size();
        int sum_of_numbers=n*(n+1)/2;
        int sum=0;
        for(int i=0;i<=n-1;i++){
            sum+=nums[i];

        }

        return sum_of_numbers-sum;

    }
};

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let n=nums.length;
    let total=0
    let sum=0;
    for(let i=0;i<n;i++){
        sum+=nums[i];
    }
    total=n*(n+1)/2


return total -sum
    
};